nuitka/plugins/standard/PlaywrightPlugin.py

```diff
@@ -127,52 +127,42 @@ def considerDataFiles(self, module):
                 "No browsers included. Use '--playwright-include-browser=browser_name' or 'all' to include them.\n"
                 "Or '--playwright-exclude-browser=browser_name' to exclude specific browsers."
             )
+
         self.getInstalledPlaywrightBrowsers()
+        available_browsers = set(self.installed_browsers.keys())
+
+        if not available_browsers:
+            self.warning("No browsers installed - nothing to include or exclude.")
+            return
 
         if "all" in self.exclude_browsers:
             self.info("All browsers excluded.")
             return
 
-        if not self.installed_browsers:
-            self.warning("No browsers installed.")
-            if self.exclude_browsers:
-                self.warning("since there are no browsers installed. Cannot exclude browsers.")
-            return
+        browsers_to_include = set(available_browsers if "all" in self.include_browsers else self.include_browsers)
```

Comment:
This looses order of the browsers and leads to random outputs and or behaviour of Nuitka